src/types/nitro.ts

```diff
@@ -208,6 +208,8 @@ export interface NitroRouteRules
   extends Omit<NitroRouteConfig, "redirect" | "cors" | "swr" | "static"> {
   redirect?: { to: string; statusCode: HTTPStatusCode };
   proxy?: { to: string } & ProxyOptions;
+  allowQuery?: string[];
+  passQuery?: boolean;
```

Comment:
Since this is a vercel-specific route rule, we might need to namespace them to as `vercel?: {}`